### PR TITLE
Issue #10: Add Basic Support for EditText

### PR DESCRIPTION
**Pull request by anesabml**

### CODE REVIEW

File: recompose-composer/src/main/kotlin/recompose/composer/writer/KotlinWriter.kt [hunk 1]
```diff
@@ -224,6 +225,21 @@ internal class KotlinWriter {
...
+            InputType.Number -> "Number"
+            InputType.Phone -> "Phone"
+            InputType.Uri -> "Uri"
+            InputType.Email -> "Email"
+            InputType.Password -> "Password"
+            InputType.NumberPassword -> "NumberPassword"
+        }
+        writer.continueLine(inputType)
+    }
+
```
Comment (anesabml) [inline recompose-composer/src/main/kotlin/recompose/composer/writer/KotlinWriter.kt:242]: Should I use Reflection to get the name of the object here instead on `when` statement?

Comment (pocmo) [reply recompose-composer/src/main/kotlin/recompose/composer/writer/KotlinWriter.kt:242]: No, I think the `when` statement is great here. :)

File: recompose-ast/src/main/kotlin/recompose/ast/view/EditTextNode.kt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ */
+
+package recompose.ast.view
+
+import recompose.ast.Node
+import recompose.ast.attributes.ViewAttributes
+import recompose.ast.values.InputType
+import recompose.visitor.Visitor
+
+data class EditTextNode(
```
Comment (pocmo) [inline recompose-ast/src/main/kotlin/recompose/ast/view/EditTextNode.kt:24]: Maybe you can add a quick kdoc here too? :)

**Review by pocmo — COMMENTED** [review]
Well done! This looks great. :)

**Review by pocmo — APPROVED** [review]

**Pull request merged**